src/controllers/UserController.js

```diff
@@ -184,6 +184,43 @@ class UserController {
       return HelperMethods.serverError(res);
     }
   }
+
+  /**
+  * Verify a user's email
+  * Route: POST: /auth/verify_email
+  * @param {object} req - HTTP Request object
+  * @param {object} res - HTTP Response object
+  * @return {res} res - HTTP Response object
+  * @memberof UserController
+ */
+  static async updatedUser(req, res) {
+    const token = req.headers['x-access-token'];
+    const { role, email } = req.body;
+    const verifiedToken = await Authentication.verifyToken(token);
+    try {
+      if (verifiedToken.role !== 'Super Administrator') {
+        return HelperMethods.clientError(res, 'Only a super admin'
+        + 'can update user role', 401);
+      }
+      const userToUpdate = await models.User.findOne({ where: { email } });
+      if (!userToUpdate) {
+        return HelperMethods.clientError(res,
+          `Cannot find user with the email: ${email}`, 404);
```

Comment:
I prefer an error message of `user not found` because it does not reveal our logic to the frontend.